SocketRocket/SRWebSocket.m

```diff
@@ -1389,6 +1401,28 @@ - (void)stream:(NSStream *)aStream handleEvent:(NSStreamEvent)eventCode;
                 });
                 return;
             }
+        } else
+        {
+            if (self.allowSelfSignedCertificates && [_urlRequest SR_SSLPinnedCertificates] == nil)
+            {
+                NSMutableArray *certs;
+                SecTrustRef secTrust = (__bridge SecTrustRef)[aStream propertyForKey:(__bridge id)kCFStreamPropertySSLPeerTrust];
+                
+                if (secTrust) {
+                    certs = [NSMutableArray array];
+                    NSInteger numCerts = SecTrustGetCertificateCount(secTrust);
+                    for (NSInteger i = 0; i < numCerts; i++) {
+                        SecCertificateRef cert = SecTrustGetCertificateAtIndex(secTrust, i);
+                        //NSData *trustedCertData = CFBridgingRelease(SecCertificateCopyData(cert));
```

Comment:
Generally it's not great form to commit commented-out code to master, since people won't know if you did it intentionally! If this was needed, let's put it back in, if it wasn't let's delete it, or if there's some good reason to leave it commented in (And there totally may be!) let's leave a comment to address that.